### ISSUE TITLE
- downloading role 'users', owned by unxnn [WARNING]: - unxnn.users was NOT installed successfully: - sorry, unxnn.users

### ISSUE THREAD

**Issue by jayargo** [NVIDIA/deepops#491]
I'm getting the following error when running deepops/scripts/setup.sh on the latest stable release (20.02)

```
% ./setup.sh 
Unsupported Operating System fedora
Please install Ansible, Git, and python-netaddr manually
fatal: destination path 'deepops' already exists and is not an empty directory.
- dev-sec.ssh-hardening (6.1.3) is already installed, skipping.
- downloading role 'users', owned by unxnn
[WARNING]: - unxnn.users was NOT installed successfully: - sorry, unxnn.users
was not found on https://galaxy.ansible.com/api/.
ERROR! - you can use --ignore-errors to skip failed roles and finish processing the list.
```

Looks like maybe unxnn.users and unxnn.chrony were deleted from ansible-galaxy, as the web link for those roles redicts to https://galaxy.ansible.com/not-found .

**Comment by ajdecon** [comment]
Thanks for the heads up! Addressing this with #492 .

**Comment by Kaka1127** [comment]
I met the similar errors like below.

```
Traceback (most recent call last):
  File "/usr/local/bin/ansible-galaxy", line 145, in <module>
    have_cli_options = cli is not None and cli.options is not None
AttributeError: 'GalaxyCLI' object has no attribute 'options'
```
Best reagrds.
Kaka

**Comment by ajdecon** [comment]
Original error should now be resolved by #492

**Comment by jayargo** [comment]
Thanks, I see this is working now!

Will the fix be put back into the 20.02 release (as I believe that release is effectively broken) or do you recommend using the latest master branch?
